ui/src/message_popup/renderers/audio_response_summary.jsx

```diff
@@ -0,0 +1,94 @@
+/* @flow weak */
+import _ from 'lodash';
+import React from 'react';
+import VelocityTransitionGroup from "velocity-react/velocity-transition-group";
+import Divider from 'material-ui/Divider';
+
+import ReadMore from './read_more.jsx';
+
+/*
+Component that displays the summary of audio responses.
+*/
+export default React.createClass({
+  displayName: 'AudioResponseSummary',
+
+  propTypes: {
+    gameSession: React.PropTypes.object.isRequired,
```

Comment:
Since `gameSession` has a lot of other things, and this component looks like it only needs the `audioResponses` list, and s it might help simplify to just pass that directly.